src/sentry/api/serializers/models/organization_member/base.py

```diff
@@ -88,6 +89,10 @@ def serialize(
             "dateCreated": obj.date_added,
             "inviteStatus": obj.get_invite_status_name(),
             "inviterName": obj.inviter.get_display_name() if obj.inviter else None,
+            "allOrgRoles": serialize(
+                obj.get_all_org_roles_sorted(),
+                serializer=OrganizationRoleSerializer(organization=obj.organization),
```

Comment:
If you're providing a list of `orgRoles` being set from multiple locations, I think you might want to extend the serializer to include their source.

Reply:
It might even make sense to sort the list by priority, so `orgRoleXyz[0]` will always be the correct org-level role for the user.

Reply:
this is currently sorted by priority, but i'll need to add the sources